app/src/main/java/com/rkpandey/mymaps/MainActivity.kt

```diff
@@ -110,27 +110,6 @@ class MainActivity : AppCompatActivity() {
 //            }
 //        }
 
-        crimeSpinner.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
```

Comment:
the problem with declaring this in your `MainActivity` is that the spinner does not actually have this spinner in its content view since it's used in the `CreateMapActivity` which shows it in a dialog. I'm not sure the intention of doing this in this activity, but if you were needing to communicate this back to another activity. You would need to do something like [this](https://stackoverflow.com/a/947560)

Reply:
Yes you're right thanks!